packages/cli/src/services/redis/redis-service-commands.ts

```diff
@@ -42,7 +43,7 @@ export type RedisServiceBaseCommand =
 	| {
 			senderId: string;
 			command: 'relay-execution-lifecycle-event';
-			payload: { type: IPushDataType; args: Record<string, unknown>; pushRef: string };
+			payload: { type: PushType; args: Record<string, unknown>; pushRef: string };
```

Comment:
Is this any `PushType` or should it be only a subset of them?

Reply:
it should definitely be a subtype, but this also requires a refactor of redis commands code first.